pytx3/setup.py

```diff
@@ -42,7 +43,7 @@
     ],
     keywords="facebook threatexchange",
     url="https://www.github.com/facebook/ThreatExchange",
-    packages=find_packages(exclude=["tests"]),
+    packages=find_packages(exclude=["tests*"]),
```

Comment:
*Author note:* This is to address removing both the top level and sub level packages within the `tests` folder in pytx3.